ansible/tests/conftest.py

```
@@ -0,0 +1,84 @@
+
+
+@pytest.fixture(scope="module")
+def host(request):
+    flake_dir = request.config.getoption("--flake-dir")
+    if not flake_dir:
+        pytest.fail("--flake-dir option is required")
+    docker_image = request.config.getoption("--docker-image")
+    if not docker_image:
+        pytest.fail("--docker-image option is required")
+    docker_id = (
+        subprocess.check_output(
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)